### PR TITLE
[Darkside] Popover CSS update

### PR DESCRIPTION
**Pull request by KenAJoh**

### CODE REVIEW

File: @navikt/core/css/darkside/popover.darkside.css [hunk 1]
```diff
@@ -1,84 +1,26 @@
...
+  border: 1px solid var(--ax-border-subtleA);
+  border-radius: var(--ax-border-radius-xlarge);
+
+  /* Creates a small gutter between screen-edges on smaller devices */
+  max-width: calc(100vw - var(--ax-spacing-6));
+
+  &:focus-visible {
+    outline: 2px solid var(--ax-border-focus);
+    outline-offset: 2px;
+  }
```
Comment (HalvorHaugan) [inline @navikt/core/css/darkside/popover.darkside.css]: How do you get focus on the popover?

File: @navikt/core/react/src/popover/Popover.tsx [hunk 2]
```diff
@@ -187,7 +190,8 @@ export const Popover = forwardRef<HTMLDivElement, PopoverProps>(
           aria-hidden={!open || !anchorEl}
         >
           {children}
-          {arrow && (
+          {/* Hide arrow in new design, prop will be remove din breaking change update */}
```
Comment (HalvorHaugan) [inline @navikt/core/react/src/popover/Popover.tsx]:
```suggestion
          {/* Hide arrow in new design, prop will be removed in breaking change update */}
```

File: @navikt/core/react/src/popover/Popover.tsx [hunk 3]
```diff
@@ -135,7 +138,7 @@ export const Popover = forwardRef<HTMLDivElement, PopoverProps>(
       placement,
       open,
       middleware: [
-        flOffset(offset ?? (arrow ? 16 : 4)),
+        flOffset(offset ?? themeContext ? 8 : arrow ? 16 : 4),
```
Comment (HalvorHaugan) [inline @navikt/core/react/src/popover/Popover.tsx]:
```suggestion
        flOffset(offset ?? (themeContext ? 8 : arrow ? 16 : 4)),
```

File: @navikt/core/css/darkside/popover.darkside.css (hunk 1 above)
Comment (KenAJoh) [reply @navikt/core/css/darkside/popover.darkside.css]: Was from the time we had `tabIndex={-1}` on Popover. Can remove it now 👍

**Review by HalvorHaugan — COMMENTED** [review]
I had some comments I forgot to submit

**Review by HalvorHaugan — APPROVED** [review]

**Pull request merged**